package.json

```diff
@@ -80,32 +80,31 @@
     "docker-toolbelt": "^3.3.7",
     "duration-js": "^4.0.0",
     "event-stream": "3.3.4",
-    "express": "^4.0.0",
+    "express": "^4.17.1",
     "fork-ts-checker-webpack-plugin": "^0.5.2",
-    "fp-ts": "^2.0.0",
-    "husky": "^4.2.1",
+    "fp-ts": "^2.0.5",
+    "husky": "^4.2.3",
     "io-ts": "^2.0.1",
     "io-ts-reporters": "^1.0.0",
     "istanbul": "^0.4.5",
     "json-mask": "^0.3.8",
-    "knex": "~0.15.2",
-    "lint-staged": "^10.0.7",
+    "knex": "^0.15.2",
+    "lint-staged": "^10.0.8",
     "livepush": "^2.0.2",
-    "lockfile": "^1.0.1",
-    "lodash": "^4.17.13",
+    "lockfile": "^1.0.4",
+    "lodash": "^4.17.15",
     "log-timestamp": "^0.1.2",
-    "memoizee": "^0.4.1",
+    "memoizee": "^0.4.14",
```

Comment:
This is not actually updated, just setting the package.json version to match the package-lock.json